docs/src/docs/asciidoc/user/getting-started.adoc

```diff
@@ -293,6 +293,27 @@ include::{sourcedir}/impl/src/test/java/org/ehcache/docs/GettingStarted.java[tag
 <3> The `DescriptionCopier` and `PersonCopier` configured at the cache manager level will be used for this cache.
 <4> The `PersonCopier` configured at the cache manager level will be used for this cache.
 
+The samples above demonstrated the configuration of different services at the cache and the cache manager levels.
+In none of the samples did we really create instances of services such as copiers, serializers etc.
```

Comment:
Yes. Yes. I'll correct that. It'd have really helped if there was some common term for these provided things.